src/frontend/sider/Journalforing.tsx

```diff
@@ -0,0 +1,140 @@
+            journalpostState.settFagsakId(fagsak.data.id);
+        }
+    }, [fagsak]);
+
+    if (!oppgaveIdParam || !journalpostIdParam) {
+        return <Redirect to="/oppgavebenk" />;
+    }
+    return (
+        <DataViewer response={journalpost}>
+            {(data: IJournalpost) => (
+                <SideLayout>
+                    <Sidetittel>{`Registrere journalpost: ${
+                        data.behandlingstema ? behandlingstemaTilTekst[data.behandlingstema] : ''
+                    }`}</Sidetittel>
+                    <Kolonner>
+                        <div>
+                            <Brukerinfo bruker={data.bruker} />
+                            <DokumentVisning
+                                journalPost={data}
+                                hentDokument={hentDokument}
+                                dokumentTitler={journalpostState.dokumentTitler}
+                                settDokumentTitler={journalpostState.settDokumentTitler}
+                            />
+                            <SkjemaGruppe
+                                feil={
+                                    journalpostState.forsøktJournalført &&
+                                    !journalpostState.behandling &&
+                                    'Du må velge en behandling for å journalføre'
+                                }
+                            >
+                                <Behandling
+                                    settBehandling={journalpostState.settBehandling}
+                                    behandling={journalpostState.behandling}
+                                    fagsak={fagsak}
+                                />
+                            </SkjemaGruppe>
+                            {journalpostState.innsending.status === RessursStatus.FEILET && (
+                                <AlertStripeFeil>
+                                    {journalpostState.innsending.frontendFeilmelding}
+                                </AlertStripeFeil>
+                            )}
+                            <FlexKnapper>
+                                <Link to="/oppgavebenk">Tilbake til oppgavebenk</Link>
+                                <Hovedknapp
+                                    onClick={() =>
+                                        journalpostState.fullførJournalføring(journalpostIdParam!)
+                                    }
+                                    spinner={
+                                        journalpostState.innsending.status === RessursStatus.HENTER
+                                    }
+                                >
+                                    Journalfør
+                                </Hovedknapp>
+                            </FlexKnapper>
+                        </div>
+                        <div>
+                            <PdfVisning pdfFilInnhold={valgtDokument} />
+                        </div>
```

Comment:
Er det nødvendig med en div-wrapper her og over for å wrappe `BrukerInfo`, `Dokumentvisning` etc. ? 
Kunne man løst det med grid evt. bare for å slippe masse div-wrappere i kildekoden?
Jeg får ikke helt til å åpne sak-frontend lokalt, så får ikke helt sett hvordan stylingen fungerer 😓

Reply:
div-wrapperne trengs for at flex skal plassere elmenetene riktig. 
Mulig grid løser behovet også, men vi vil fremdeles behøve en form for wrapper - da må den bare spesifiseres tydeliggere?